README.md

```diff
@@ -337,5 +326,87 @@ Contact the dev team for the following:
 
 Once you have access to AWS LZ2 platform, login to the any of the environments to get access to the following: 
 
+- Project .env file can be found in the prod parameter store under pcc/.env
 - Database passwords
 - AWS Local Config - Service Account Tokens
+
+
+## Deployments
+
+Command Reference: 
+
+Build backend:
+```bash
+make build-backend
+```
+
+Terraform:
+```bash
+make plan
+```
+
+```bash
+make apply
+```
+Tag Based Deployments:
+```bash
+make tag-<env>
+```
+
+Deployments and builds are initiated via GitHub actions.
+We have four environments, each with a corresponding GHA yaml file:
```

Comment:
We have four environments: dev, tools, test, and prod - with an associated GitHub Actions YAML file for each.